spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -178,53 +178,29 @@
   end
 
   context 'when not logged in' do
-    describe ' GET new' do
-      it 'creates the saml authn request with LOA 1 if supplied level=1' do
-        get :new, level: 1
-        expect(SAML::AuthnRequestHelper.new(response).loa1?).to eq(true)
-      end
-
-      it 'returns the urls for for all three possible authN requests' do
-        get :authn_urls
-        expect(response).to have_http_status(200)
-        expect(JSON.parse(response.body).keys).to eq %w(mhv dslogon idme)
-      end
-
-      it 'returns does not allow fetching the identity proof url' do
-        get :identity_proof
-        expect(response).to have_http_status(401)
-      end
+    it 'returns the urls for for all three possible authN requests' do
+      get :authn_urls
+      expect(response).to have_http_status(200)
+      expect(JSON.parse(response.body).keys).to eq %w(mhv dslogon idme)
+    end
 
-      it 'does not allow fetching the multifactor url' do
-        get :multifactor
-        expect(response).to have_http_status(401)
-      end
+    it 'returns does not allow fetching the identity proof url' do
```

Comment:
I don't think the 'returns' is supposed to be there.

Reply:
good catch.